pandas/plotting/_core.py

```diff
@@ -2121,6 +2110,73 @@ def plot_series(data, kind='line', ax=None,                    # Series unique
         :context: close-figs
 
         >>> boxplot = df.boxplot(grid=False, rot=45, fontsize=15)
+
+
+    The parameter ``return_type`` can be used to select the type of element
+    returned by `boxplot`.  When ``return_type='axes'`` is selected,
+    the matplotlib axes on which the boxplot is drawn are returned:
+
+        >>> df.boxplot(column=['Col1','Col2'], return_type='axes')
```

Comment:
IMHO it might be enough with a couple of examples for the return types, perhaps being so exhaustive is too verbose and a bit overkill, since one has to care about the ellipsis and all... The different possibilities are already explained at the parameters section.

Reply:
OK. I'll keep just the most complicated case that needed explanation from the parameters definition.

Reply:
FYI, you can do `axes = df.boxplot(...)`. Then the return isn't printed out and the test will pass.

Reply:
If you want to make a point about the return type, you can do 

```
>>> result = df.boxplot(...)
>>> type(result)
<class pandas.Series>
````

or something like that.

Reply:
Great tip! Didn't know how to do it. The internal structure is not shown but it's enough. Thanks!